test/Dialect/ESI/connectivity.mlir

```diff
@@ -64,23 +64,23 @@ hw.module @testIfaceWrap() {
   %idataChanOut = esi.wrap.iface %ifaceOutSink: !sv.modport<@IData::@Sink> -> !esi.channel<i32>
 
   // CHECK-LABEL:  hw.module @testIfaceWrap() {
-  // CHECK-NEXT:     %0 = sv.interface.instance {name = "ifaceOut"} : !sv.interface<@IData>
-  // CHECK-NEXT:     %1 = sv.modport.get %0 @Source : !sv.interface<@IData> -> !sv.modport<@IData::@Source>
-  // CHECK-NEXT:     hw.instance "ifaceSender" @IFaceSender(x: %1: !sv.modport<@IData::@Source>) -> ()
-  // CHECK-NEXT:     %2 = sv.modport.get %0 @Sink : !sv.interface<@IData> -> !sv.modport<@IData::@Sink>
-  // CHECK-NEXT:     %3 = esi.wrap.iface %2 : !sv.modport<@IData::@Sink> -> !esi.channel<i32>
+  // CHECK-NEXT:     %ifaceOut = sv.interface.instance : !sv.interface<@IData>
+  // CHECK-NEXT:     %0 = sv.modport.get %ifaceOut @Source : !sv.interface<@IData> -> !sv.modport<@IData::@Source>
+  // CHECK-NEXT:     hw.instance "ifaceSender" @IFaceSender(x: %0: !sv.modport<@IData::@Source>) -> ()
+  // CHECK-NEXT:     %1 = sv.modport.get %ifaceOut @Sink : !sv.interface<@IData> -> !sv.modport<@IData::@Sink>
+  // CHECK-NEXT:     %2 = esi.wrap.iface %1 : !sv.modport<@IData::@Sink> -> !esi.channel<i32>
```

Comment:
The numeric matches were already brittle before this PR. 😬 Consider cleaning these up to use captures to avoid this going forward, e.g., `%[[#modport_0:]]` is the tersest way I can find to do this. (I would normally do this with `%[[modport_0:[0-9]+]]`, but I just found the other approach.)

This is fine to do in this PR or in another NFC PR/commit.